ocs_ci/deployment/deployment.py

```diff
@@ -398,8 +398,10 @@ def subscribe_ocs(self):
         time.sleep(90)
         if subscription_plan_approval == "Manual":
             wait_for_install_plan_and_approve(self.namespace)
-            logger.info("Sleeping for 60 seconds after approving install plan for OCS")
-            time.sleep(60)
+            channel = config.DEPLOYMENT.get("ocs_csv_channel")
```

Comment:
can we use variable https://github.com/red-hat-storage/ocs-ci/pull/5304/files#diff-8571fc52f1040ac2cb01985da6d61cc93a535f812814c200b45d00daf28c2e3fR377